src/common/wui_api.c

```diff
@@ -6,6 +6,43 @@
  *      Author: joshy <joshymjose[at]gmail.com>
  */
 #include "wui_api.h"
+#include "version.h"
+#include "otp.h"
+#include <string.h>
+#include <stdio.h>
+
+#define PRINTER_TYPE_ADDR       0x0802002F  // 1 B
```

Comment:
should not redefine these address, might be already defined somewhere?

Reply:
I didn't find it